README.md

```diff
@@ -81,6 +81,18 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
 
 ## Installation
 
+### Any OS (development environment)
+
+1. Download Yarn and Node.js
+2. Download the dependencies with `yarn`
+3. Start Heroic by pressing F5 on vs code or `yarn react-start && yarn electron`
```

Comment:
In this case, pressing F5 will run only the react process? or will run both commands?

Reply:
It will run both commands, see .vscode/launch.json

Reply:
yes it is only running the first configuration working on a fix rn